common/components/ui/Modal.scss

```diff
@@ -117,7 +121,6 @@ $m-anim-speed: 400ms;
   }
 
   &-footer {
-    height: $m-footer-height;
```

Comment:
It's possible this was here for a reason, was it causing you any problems?

Reply:
nope, seems I just removed it by mistake.

Reply:
nvm, I double checked and it was causing a problem. Setting the height manually defeats the purpose of having standardized buttons that define the height of the modal footer. Additionally, setting the height this way reduces the height of the modal footer, making the padding mis-aligned

https://imgur.com/a/VdiHS

Reply:
👍